docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,109 @@
+# Tutorial: Deploy the Penpot charm
+
+## What you'll do
+This tutorial will walk you through deploying the penpot charm; you will:
+1. Deploy and configure the Penpot charm's dependencies (PostgreSQL, Redis, s3, etc.)
+2. Deploy Penpot and integrate it with its dependencies
+3. Create a user
+4. Login to the web interface for your deployed Penpot instance
+
+## Prerequisites
+* MicroK8s deployed with the `ingress`, `dns`, and `storage` plugins enabled.
+* A host machine with Juju version 3.4 or above and a Juju controller bootstrapped.
+* The `boto3` python package installed - `pip install boto3`.
+
+## Create a Juju model
+1. Create a Juju model
+```
+juju add-model penpot-test
+```
+
+## Deploy and configure the Penpot charm's dependencies
+1. Deploy and configure some dependencies
+```
+juju deploy minio --config access-key=minioadmin --config secret-key=minioadmin
+juju deploy postgresql-k8s --channel 14/stable --trust
+juju deploy redis-k8s --channel latest/edge
+juju deploy s3-integrator --config "endpoint=http://minio-endpoints.penpot-test.svc.cluster.local:9000" --config bucket=penpot
+juju deploy nginx-ingress-integrator --trust --config service-hostname=penpot.local --config path-routes=/
+juju deploy self-signed-certificates
+```
+
+## Configure Minio
+1. Run `juju status` to confirm the `minio/0` unit is `active/idle` (may take a few minutes), create the `penpot` bucket:
+```
```

Comment:
As an arguably simpler alternative, a client like `aws-cli` or `s3cmd` could be used, like:
```
sudo snap install aws-cli --classic
export AWS_ACCESS_KEY_ID=minioadmin
export AWS_SECRET_ACCESS_KEY=minioadmin
export AWS_ENDPOINT_URL=http://$(juju status --format=json | jq -r '.applications.minio.units."minio/0".address'):9000
aws s3 mb s3://penpot
```

Reply:
Like it, will update.